src/libpcp_web/src/webgroup.c

```diff
@@ -1911,8 +1933,10 @@ webgroup_scrape_batch(const char *name, void *arg)
 	scrape->numnames = 0;
     }
 
-    scrape->names[scrape->numnames] = sdsnew(name);
-    scrape->numnames++;
+    if(!filtered(scrape, name)) {
```

Comment:
Missing whitespace after "if" here.

Reply:
I rebased on the current upstream master branch and updated the patches to address the comments above.  It should be ready to merge now.